### ISSUE TITLE
Use default settings when nestcli.json is missing

### ISSUE THREAD

**Issue by kamilmysliwiec** [nestjs/nest-cli#174]
When `nestcli.json` doesn't exist, use defaults instead of throwing an error. Related: https://github.com/nestjs/typescript-starter/issues/50

Also, consider renaming to `nest-cli.json`, but to stay backward-compatible, use `nestcli.json` when `nest-cli.json` is not available.

**Comment by walterspieler** [comment]
I struggled a bit to find the default content of .nestcli.json file and had to generate a new nest project to get it.
The content of default .nestcli.json (That might help)
```
{
  "language": "ts",
  "collection": "@nestjs/schematics"
}
```

**Comment by kamilmysliwiec** [comment]
solved by #176
